packages/xforms-engine/src/lib/NumberParsers.ts

```diff
@@ -0,0 +1,25 @@
+export const parseToInteger = (value: string | null): number | null => {
+	if (value === null) {
+		return null;
+	}
+
+	const parsed = Number(value);
+	if (typeof value !== 'string' || value.trim() === '' || !Number.isInteger(parsed)) {
+		throw new Error(`Expected an integer, but got: ${value}`);
+	}
+
+	return parsed;
+};
+
+export const parseToFloat = (value: string | null): number | null => {
+	if (value === null) {
+		return null;
+	}
+
+	const parsed = Number(value);
+	if (typeof value !== 'string' || value.trim() === '' || isNaN(parsed)) {
+		throw new Error(`Expected a float, but got: ${value}`);
+	}
+
+	return parsed;
+};
```

Comment:
I don't want to send this through another round trip, just for future reference: we usually use `PascalCase` modules names as 1:1 with a class and/or interface of the same name. If I were creating this module I'd probably `kebab-case` it.

Reply:
I also see a couple of filenames with `cammelCase`